grails-app/conf/BuildConfig.groovy

```diff
@@ -67,10 +68,6 @@ grails.project.dependency.resolution = {
         runtime ":database-migration:1.4.0"
         runtime ":jquery:1.11.1"
 
-        // Uncomment these to enable additional asset-pipeline capabilities
-        //compile ":sass-asset-pipeline:1.9.0"
-        //compile ":less-asset-pipeline:1.10.0"
-        //compile ":coffee-asset-pipeline:1.8.0"
-        //compile ":handlebars-asset-pipeline:1.3.0.3"
+        build ":tomcat:$grailsVersion"
```

Comment:
I think for portal I had to use here something like 7.0.xx. Seems like 2.4.4 might be too old (or non-existent).

Are you running it with `grails run-app`?